project/script_IR.tcl

```diff
@@ -27,10 +27,10 @@ create_clock -period 240MHz -name slow_clock
 create_clock -period 360MHz -name fast_clock
 
 set nProc [exec nproc]
-csim_design -mflags "-j$nProc" -argv "--link,8 --dtcSplit,0 --tkNonant,4"
+csim_design -compiler gcc -mflags "-j$nProc" -argv "--link,8 --dtcSplit,0 --tkNonant,4 --lastBx,100"
```

Comment:
Apparently we're not supposed to use the compiler flag anymore due to Vitis compatibility (quote Andrew)

Reply:
thanks - didn't catch that I'd changed it back. will fix.